src/bittytax/conv/parsers/nexo.py

```diff
@@ -77,8 +77,16 @@ def parse_nexo(data_row: "DataRow", parser: DataParser, **_kwargs: Unpack[Parser
         value = None
 
     if row_dict["Type"] in ("Deposit", "Top up Crypto", "ExchangeDepositedOn"):
+        # Skip credit deposits (already handled with "Loan Withdrawal").
+        if row_dict["Details"].find("Credit") > -1:
```

Comment:
Do you have an example row for this "Credit" deposit? Is this you providing collateral for the loan?

Reply:
Nexo creates two transactions for a loan, a "Loan Withdrawal" and this deposit ("Top up Crypto"). We only need one, so I found it easier to ignore the top up since they add a note indicating it's a credit they've granted. I'll compile something with examples from my file.

Reply:
Example attached.
[nexo_transactions_example.csv](https://github.com/BittyTax/BittyTax/files/13706270/nexo_transactions_example.csv)